src/pytom_tm/tmjob.py

```diff
@@ -61,6 +61,13 @@ def load_json_to_tmjob(file_name: pathlib.Path, load_for_extraction: bool = True
         pytom_tm_version_number=data.get('pytom_tm_version_number', '0.3.0'),
         job_loaded_for_extraction=load_for_extraction,
     )
+    # if the file originates from an old version set the phase shift for compatibility
+    if (
+        version.parse(job.pytom_tm_version_number) < version.parse('0.6.1') and
+        job.ctf_data is not None
+    ):
+        for tilt in job.ctf_data:
+            tilt['phase_shift_deg'] = .0
```

Comment:
This is not hit in the tests, can you add a test to make sure this compatibility check works as expected?

Reply:
I just now added a job file from before. Had to change the file paths in the job to accommodate to point to the dynamically generated test files.